aea/helpers/storage/backends/binaries/README.txt

```diff
@@ -0,0 +1 @@
+json1.dll - is sqlite extension for windows python<3.9
```

Comment:
you need to make sure `.dll` is included in the MANIFEST.in so this gets distributed properly